tests/dummy_triton_models/multimodal-embedder/1/model.py

```diff
@@ -53,22 +46,30 @@ def execute(self, requests):
     responses = []
 
     for request in requests:
-      parameters = request.parameters()
+      try:
+        parameters = request.parameters()
+      except Exception:
+        print(
```

Comment:
> disclaimer: I'm Sumit, looking forward to contribute to Clarifai's open source, get involved with the team for also plausible employment in Tech at Clarifai, kindly excuse my 2 cents and contributions hither and thither (=:

Instead, of mere "print"s, may be benefit from using a proper `logging` library and a configuration? We can begin logging such values further, more appropriately by suitable Log Levels like WARN, ERROR, CRITICAL, INFO etc.